README.md

```diff
@@ -24,7 +24,7 @@ To install RSpec, run the following command in your terminal:
 
     gem install rspec
 
-You should see a message saying "5 gems installed" in your terminal. Verify that the installation was successful by simply running the following command:
+You should see a message saying "6 gems installed" in your terminal. Verify that the installation was successful by simply running the following command:
```

Comment:
When I test this locally, I saw a different number. The actual number doesn't matter, so what about changing the line to:
```suggestion
You should see a message that says something along the lines of "6 gems installed" in your terminal. Verify that the installation was successful by simply running the following command:
```

Reply:
makes sense. Thank you